aws-auth-cognito/src/main/java/com/amplifyframework/auth/cognito/RealAWSCognitoAuthPlugin.kt

```diff
@@ -1507,7 +1562,12 @@ internal class RealAWSCognitoAuthPlugin(
 
                 when (authState) {
                     is AuthState.WaitingForCachedCredentials -> credentialStoreStateMachine.send(
-                        CredentialStoreEvent(CredentialStoreEvent.EventType.LoadCredentialStore())
+                        CredentialStoreEvent(
+                            CredentialStoreEvent.EventType.LoadCredentialStore(
+                                AmplifyCredentialType.DEVICE_METADATA,
+                                null
```

Comment:
nit: rather than passing null as a parameter, how about setting default value of param to `null` if it is nullable?

Reply:
We removed the nullable property and now using sealed data classes with properties.